Framework/Core/include/Framework/DataAllocator.h

```diff
@@ -41,7 +41,7 @@
 #include <cstddef>
 
 // Do not change this for a full inclusion of FairMQDevice.
-class FairMQDevice;
+#include <FairMQDevice.h>
```

Comment:
Is this really necessary? There was a big cost in including the `FairMQDevice.h` due to the boost state machine.

Reply:
FairMQ headers do not contain MSM anymore, all MSM is in source files.

Reply:
Is it true? I see 

```C++
#include <FairMQStateMachine.h>
```

which itself includes MSM. When was this changed?

Reply:
In May 2018 MSM includes were moved to FairMQStateMachine.cxx.